modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/notifications/ScheduledPublicationUserNotificationHandler.java

```diff
@@ -61,7 +62,10 @@ protected String getBody(
 				_language.format(
 					serviceContext.getLocale(),
 					"x-scheduled-publication-failed",
-					new Object[] {jsonObject.getString("ctCollectionName")},
+					new Object[] {
+						HtmlUtil.escape(
+							jsonObject.getString("ctCollectionName"))
```

Comment:
Do we need to escape this twice? You're already escaping it in `CTPublishBackgroundTaskExecutor`.

Reply:
I went through looking to escape instances of the ctCollectionName being used in notification-related strings. It seemed like this is separate from `CTPublishBackgroundTaskExecutor`. Does data from `ScheduledPublicationUserNotificationHandler` get passed through `CTPublishBackgroundTaskExecutor` at some point that I'm missing?